### ISSUE TITLE
NTR: Dimethyl sterols high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8393]
New term request Dimethyl sterols high in blood (elthat_160318115436):

**Comment by pnrobinson** [comment]
@MickeySegal
are you referring to 4,4-Dimethylsterols (see https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6385002)?

**Comment by MickeySegal** [comment]
That seems to be what is referred to in https://www.jci.org/articles/view/42650 "The levels of dimethylsterols and monomethylsterols were markedly increased in patient skin. Dimethylsterol (peak 6) is most elevated in skin, suggesting the preferential accumulation of 4,4′-dimethylsterols in the patient’s skin."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000754
- label: Elevated circulating dimethyl sterol concentration
- parents: [Abnormal circulating lipid concentration (HP:0003119)](https://hpo.jax.org/app/browse/term/HP:0003119)
- pmid: PMID:21285510
- def: The concentration of dimethylsterols in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8393
- synonyms:
